### DISCUSSION TITLE
v2.20.70 - 5.3K Resolution Short clips

### DISCUSSION THREAD

**Discussion by DVGPR0** [gopro/labs#340] [General]
With the latest firmware, I record a clip longer than 90 seconds at 5.3K and 60 FPS. GoPro Support thinks it’s an overheating issue but it’s not _that_ hot and it doesn’t give me that error. Any help would be appreciated.

**Comment by dnewman-gpsw** [comment]
Any other Labs features running?  Are you increasing the bitrate, as that can fail easier at 60fps+. I just tried without issue, setting on my desk, still going fine at 10 minutes when I stopped recording (running at standard 60Mbs/s).

**Reply by DVGPR0** [reply]
Yeah, I had the bitrate set to 200. I brought it down to 160 and so far it seems okay. Thank you for the reply. Big fan of labs.
